src/features/courses/course-route.ts

```diff
@@ -1640,4 +1640,18 @@ router.post('/question/editor/catalog',
             problems: Object.keys(result).filter(elm => elm.endsWith('.pg'))
         }));
     }));
+
+router.post('/feedback', 
+    authenticationMiddleware,
+    validate(postFeedbackValidation),
+    asyncHandler(async (req: RederlyExpressRequest<PostFeedbackRequest.params, unknown, PostFeedbackRequest.body, unknown>, _res: Response, next: NextFunction) => {
+        const res = courseController.addFeedback({
```

Comment:
your not awaiting here, so no error handling and the response is probably garbage